core/node/fee_model/src/l1_gas_price/gas_adjuster/mod.rs

```diff
@@ -166,8 +177,9 @@ impl GasAdjuster {
         let calculated_price =
             (self.config.internal_l1_pricing_multiplier * effective_gas_price as f64) as u64;
 
-        // Bound the price if it's too high.
-        self.bound_gas_price(calculated_price)
+        let conversion_rate = self.base_token_fetcher.conversion_rate().unwrap_or(1);
```

Comment:
What about blobs? Seems like the gas prices there aren't determined using this.

Reply:
The `estimate_effective_gas_price` function is not concerned with the whole blob/pubdata pricing stuff, that's done by the `estimate_effective_pubdata_price` function. That being said, because this PR was done before blob functionality was introduced, that last function seems to be outdated and the price it returns should also be multiplied by the conversion rate on the `PubdataSendingMode::Blobs` case (see [here](https://github.com/matter-labs/zksync-era/blob/1e0a0d4cf6b74ffc87b8e044a3b258a6410eb4f5/core/node/fee_model/src/l1_gas_price/gas_adjuster/mod.rs#L185))

cc @popzxc for visiblity

Reply:
Exactly. This has to be a part of this PR otherwise it doesn't make sense. @jrchatruc can you please fix that?

Reply:
Fixed it
```rs
    ...
    let conversion_rate = 
    self.base_token_fetcher.conversion_rate().unwrap_or(1);

    self.bound_blob_base_fee(calculated_price) * conversion_rate
}
```

Reply:
thanks! there's going to be a collision now with the other PR so just note that this is kept it with the new Decimal